### PR TITLE
Add landing page animation feature

### LINKED ISSUE #311: Add Anitmation to feature list after 0.7 is released
As the title says, we should add a new entry in the feature list for animations. I assume cart will have a screenshot or gif in the 0.7 blog post so we could probably just reuse that.

### PR DESCRIPTION
**Pull request by TrialDragon**
Add a landing page description for animation feature.

Closes #311 

TODO:
- [x] Add screenshot---or ideally gif---to the feature description. (The 0.7 news stuff would work, but is mp4, not sure if allowed to turn one of the examples into gif for use or any of that).
- [x] Add an accurate feature description. (I guess ask in the animation related channels on discord, or hope someone with more knowledge provides some useful info here.)

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]

**Review by matiqo15 — APPROVED** [review]

File: static/assets/fox.png
Comment (mockersf) [inline static/assets/fox.png:1]: This file is too large

Comment (TrialDragon) [reply static/assets/fox.png:1]: How much smaller does it need to be? I can try to lower resolution, but that may only reduce it to like 1mb or just a bit dmaller. I could also turn it into a gif, but then it looks like jagged compressed palette hell and not a good look on the landing page imo.

Comment (TrialDragon) [reply static/assets/fox.png:1]: Okay, lowering resolution made the animated png approx 600kb about half the size it was.

Comment (mockersf) [reply static/assets/fox.png:1]: Rule of thumbs is no file bigger than 500kb on the welcome page of a website, I prefer to limit to 200kb but that can be hard for an animation...

Could you try with the webp format? It should be the best for file size

Comment (TrialDragon) [reply static/assets/fox.png:1]: That would reduce it to 300kb, yes. Will do the change tomorrow when at PC with access to repo.

**Comment by cart** [comment]
Just pushed some small tweaks / added a bullet point about animation blending. Great work!

**Review by cart — APPROVED** [review]
I'll resolve merge conflicts / get this merged!

**Pull request merged**